tests/acceptance/blocking_adapter_test.py

```diff
@@ -208,8 +208,43 @@ def test(self):
         # Publish the message to the queue by way of default exchange
         ch.publish(exchange='', routing_key=q_name, body=body1)
 
-        # Create a non-ackable consumer
-        ch.basic_consume(q_name, lambda *x: None, no_ack=True,
+        # Create a consumer that uses automatic ack mode
+        ch.basic_consume(q_name, lambda *x: None, auto_ack=True,
+                         exclusive=False, arguments=None)
+
+        connection.close()
+        self.assertTrue(connection.is_closed)
+        self.assertFalse(connection.is_open)
+        self.assertFalse(connection.is_closing)
+
+        self.assertFalse(connection._impl._channels)
+
+        self.assertFalse(ch._consumer_infos)
+        self.assertFalse(ch._impl._consumers)
+
+class TestCreateAndCloseConnectionWithChannelAndConsumerUsingLegacyAcknowledgementModeKeyword(BlockingTestCaseBase):
```

Comment:
I don't think we need this test any more since legacy keyword is gone and this tests looks pretty much like the one above it